mathesar_ui/src/systems/table-view/header/header-cell/ColumnHeaderContextMenu.svelte

```diff
@@ -51,6 +57,12 @@
   function removeGrouping() {
     grouping.update((g) => g.withoutColumns([columnId]));
   }
+
+  function goToTable() {
```

Comment:
Remove this function. We won't need it when we change the `ButtonMenuItem` instance to a `LinkMenuItem`.

Doing things like `window.location.href = yadaYada` _works_ but is not great. With a `LinkMenuItem` we get an actual hyperlink for the user to click on. That's better because it allows them to do things like copy the link or open the link in a new tab.